File: app/src/main/java/com/example/rlapp/ui/payment/Price.java
package com.example.rlapp.ui.payment;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Price {

    @SerializedName("INR")
    @Expose
    private Integer inr;
    @SerializedName("USD")
    @Expose
    private Integer usd;

    public Integer getInr() {
        return inr;
    }

    public void setInr(Integer inr) {
        this.inr = inr;
    }

    public Integer getUsd() {
        return usd;
    }

    public void setUsd(Integer usd) {
        this.usd = usd;
    }

}

File: app/src/main/java/com/jetsynthesys/rightlife/subscriptions/pojo/Price.kt
package com.jetsynthesys.rightlife.subscriptions.pojo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Price {
    @SerializedName("INR")
    @Expose
    var inr: Int? = null

    @SerializedName("USD")
    @Expose
    var usd: Double? = null
}